enterprise-suite/es-monitor-api/default-monitors.json

```diff
@@ -104,6 +104,24 @@
           "description": "{{$labels.instance}} has anomalous scrape_duration_seconds"
         }
       },
+      "es_monitor_api_errors": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "es_monitor_api_last_error",
+          "window": "1m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "es console monitor errors",
+          "description": "one or more monitors are erroneous"
```

Comment:
"one or more monitors are erroneous" seems an odd phrasing and possibly not correct if the issue is something not in the parsing of the monitor. I think we want to direct attention to logs for specifics, so I would maybe put the instance information in here

"Lightbend monitor api on {{$labels.instance}} has errors"